mxnet/buildspec.yml

```diff
@@ -68,33 +68,6 @@ images:
     target: e3
     context:
       <<: *TRAINING_CONTEXT
```

Comment:
nit: Should we rename the file to add `-ec2` in it so that the usage is clear?

Reply:
We could, but it needs an infrastructure level change as well to accompany that.